### PR TITLE
Updated qTesla with round2 implementation

### PR DESCRIPTION
**Pull request by christianpaquin**
Updated qTesla with round2 implementation. In so doing, removed the heuristic variants and added the provably secure ones.

### CODE REVIEW

**Comment by xvzcf** [comment]
Could you also update the OpenSSL and OpenSSH forks with the new signatures?

**Comment by christianpaquin** [comment]
@xvzcf, I'll work on that next.

File: VisualStudio/oqs/oqs.vcxproj [hunk 1]
```diff
@@ -43,6 +43,7 @@
     <ClInclude Include="..\..\src\kem\saber\kem_saber.h" />
     <ClInclude Include="..\..\src\sig\dilithium\sig_dilithium.h" />
     <ClInclude Include="..\..\src\sig\mqdss\sig_mqdss.h" />
+    <ClInclude Include="..\..\src\sig\qtesla\sig_qtesla.h" />
```
Comment (dstebila) [inline VisualStudio/oqs/oqs.vcxproj]: This line needs to go outside the auto-generated code block

File: VisualStudio/oqs/oqs.vcxproj.filters [hunk 2]
```diff
@@ -1905,6 +1917,9 @@
     <Filter Include="sphincs\shake256_256s_simple">
       <UniqueIdentifier>{a7c0f5cd-c1e4-4880-9c9a-98d0b09d03ab}</UniqueIdentifier>
     </Filter>
+    <Filter Include="qtesla">
```
Comment (dstebila) [inline VisualStudio/oqs/oqs.vcxproj.filters]: These need to go outside the autogenerated code block

**Pull request merged**